test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1872,6 +1872,27 @@ items:
     - get
     - list
   - apiGroups:
+    - extensions
+    resources:
+    - daemonsets
+    verbs:
+    - get
+    - list
+  - apiGroups:
+    - extensions
+    resources:
+    - deployments
+    verbs:
+    - get
+    - list
+  - apiGroups:
+    - extensions
+    resources:
+    - replicasets
+    verbs:
+    - get
+    - list
+  - apiGroups:
```

Comment:
have these been added to the image-pruner role?

Reply:
> have these been added to the image-pruner role?

Yes.